components/scream/docs/build.md

```diff
@@ -0,0 +1,93 @@
+# Building and Testing Standalone SCREAM
+
+Follow these simple instructions to build and test SCREAM's standalone
+configuration for yourself. This document makes use of the following
+paths:
+
++ `${RUN_ROOT_DIR}`: the root directory where SCREAM is built and run
++ `${SCREAM_SRC_DIR}`: the directory into which you've cloned the `scream` repo
+
+## 1. Start From a Trustworthy Commit
+
+First, make sure you've cloned the [SCREAM repo](https://github.com/E3SM-Project/scream)
+to `SCREAM_SRC_DIR`. If you're running a branch that's not `master`, check out
+this branch with
+
+```
+git checkout <branch>
+```
+
+## 2. Configure Your SCREAM Build
+
+Create a test directory in `RUN_ROOT_DIR` and use CMake to configure your build.
+This usually looks something like the following:
+```
+mkdir -p $RUN_ROOT_DIR/test
+cmake \
+    -D CMAKE_INSTALL_PREFIX=${RUN_ROOT_DIR}/install \
```

Comment:
Do we support installation? I would stay away from install related stuff.

Reply:
Removed.